src/main/java/seedu/duke/wordlist/WordList.java

```diff
@@ -22,7 +22,7 @@ public static int getNumberOfWords() {
      */
     public static void addNoun(String input) {
         String[] word = input.split(" ", 2);
-        String[] splitInput = word[1].split("d/");
+        String[] splitInput = word[1].split("d\\\\");
```

Comment:
Hi Yuxuan should line 25 be "String[] splitInput = word[1].split("d\\");" instead of "d\\\\"? We only use one backslash symbol.